source/install/transport-encryption/config-cluster.rst

```diff
@@ -0,0 +1,231 @@
+=============================================
+Configuring Cluster Transport Encryption (E20)
+=============================================
+
+Mattermost is able to encrypt the messages sent within the cluster of a deployment
+using SSH tunneling. The guide walks through the deployment of this solution on
+Ubuntu 18.04, but it can be adapted for any Linux operating system.
+
+While this document only describes the configuration of a three-node cluster, it is
+by no means limited to that amount.
+
+## Prerequisites
+- SSH port whitelisted between each node of the deployment
```

Comment:
Do we need to provide a port number? Is this done on Mattermost?

Reply:
No, this is usually a standard port that may depend on the persons configuration as well. We can safely assume they know this.